packages/app/lib/io/p2panda/node.dart

```diff
@@ -6,6 +6,10 @@ import 'package:app/io/p2panda/key_pair.dart';
 import 'package:app/io/p2panda/p2panda.dart';
 import 'package:app/utils/sleep.dart';
 
+const List<String> relayAddresses = bool.hasEnvironment("RELAY_ADDRESS")
```

Comment:
I easily find a way to pass a list in via environment vars and I thought this was good enough for now.

Reply:
Think this is a good solution!